server/routes/question.js

```diff
@@ -1,5 +1,119 @@
 var express = require('express');
 var router = express.Router();
+var Survey = require('./../schemas/survey');
 
+//////////////////////////////////////
+///////////question Routes
+//////////////////////////////////////
+
+
+//CREATE question route: Adds a new question to the collection (POST)
+router.post('/questions/:surveyID',  async (req, res, next) => {
+  console.log("in /questions route (POST) with params = " + JSON.stringify(req.params) +
+    " and body = " + JSON.stringify(req.body));  
+    if (!req.body.hasOwnProperty("questionID") || 
+    !req.body.hasOwnProperty("questionID") || 
+    !req.body.hasOwnProperty("questionTitle") || 
+    !req.body.hasOwnProperty("questionText") ||
+    !req.body.hasOwnProperty("questionType") ||
+    !req.body.hasOwnProperty("questionAnswers") || 
+    !req.body.hasOwnProperty("questionActive"))
+    {
+        return res.status(400).sent("POST request on /questions formulated incorrectly." +
+        "Body must contain all 6 required fields, questionID, quiestionTitle, questionText, questionType, questionAnswers, questionActive");
+    }
+  try {
+      thisSurvey = await Survey.updateOne(
+        {surveyID: req.params.surveyID},
+        {$push: {questions: req.body}});
```

Comment:
Was this tested and confirmed that it does the intended action? @Danejerosa1111

Reply:
Code wise it looks good, if this is confirmed to work as Osman has mentioned then I will approve of the request.

Reply:
Already discussed about this. Good to go.